convert-to-json.py

```diff
@@ -15,22 +15,23 @@
 
 system_name = os.path.basename(os.path.realpath(folder_name))
 
-with open(os.path.join(folder_name, "scf.in")) as fhandle:
+with open(os.path.join(folder_name, "GaAs.scf.in")) as fhandle:
     scf_input = fhandle.read()
-with open(os.path.join(folder_name, "scf.out")) as fhandle:
+with open(os.path.join(folder_name, "GaAs.scf.out")) as fhandle:
     scf_output = fhandle.read()
-with open(os.path.join(folder_name, "matdyn.modes")) as fhandle:
+with open(os.path.join(folder_name, "GaAs.modes")) as fhandle:
     matdyn_modes = fhandle.read()
 
-pretty_name_dict = {"BaTiO_3": "BaTiO<sub>3</sub>"}
+pretty_name_dict = {"GaAs": "GaAs"}
 
-highsym_qpts_default = [[0, "Γ"], [20, "M"], [40, "K"], [60, "Γ"]]
+highsym_qpts_default = [[0, "L"], [40, "Γ"], [80, "K"], [100, "X"], [140, "Γ"]]
```

Comment:
Here, this is the default qpts, that was the same for all the 2D materials. Before, the only expection was BaTiO3. I think this should be kept as it was, unless there is a better default.